### ISSUE TITLE
[Bug]: Anvil applications not working for unknown reasons

### ISSUE THREAD

**Issue by Shibva** [Fuzss/easyanvils#29]
### Mod Loader (Required)

Fabric

### Minecraft Version (Required)

1.19.2

### Mod Version (Required)

4.0.11

### Notes (Required)

other aspects of the mod are working but for some reason the anvil is not working

This may be due to some other mods possible overwriting the changes that are trying to be implimented. My guess is that it has todo wtih Zenith (Farbic Fork for Apotheosis thats a WIP) or BetterEnd that impliments a means to cycle crafting (if this is whats causing it then a means to have it work with it may need to be implemented to prevent crafting issues)

### latest.log (Optional)

https://gist.github.com/Shibva/53640a5270a9c94bd8bcdf9c55f98114

**Comment by Shibva** [comment]
UPDATE: the issue is caused by Zenith, the fabric equivalence (unofficial WIP fork) of Apotheosis

the way it handles its anvils apparently writes over what is implemented here

**Comment by Fuzss** [comment]
Yeah, Easy Anvils is not compatible with Zenith. This likely won't change in the future.

**Comment by bastirlnd** [comment]
Potential workaround I tested earlier, cause I couldn't let this go:
https://github.com/TheWinABagel/Zenith/pull/234
